benchmarks/01_trivial/14_shift.cpp

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) 2025, Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <benchmark/benchmark.h>
+#include <cmath>
+#include <cstdint>
+#include <stdio.h>
+
+#include "../common.h"
+#include "14_shift_ispc.h"
+
+static Docs docs("shift_const_id - test basic shift functionality with offset -1\n"
+                 "shift_<type> - test shift functionality for different data types:\n"
+                 "  int8, int16, int32, int64, uint8, uint16, uint32, uint64, float16, float, double\n"
```

Comment:
There is no benchmarks for unsigned types so either comment or the benchmarks itself should be updated.

Reply:
Right, this is copy-paste leftovers.